moderator/settings.py

```diff
@@ -220,6 +220,7 @@ def _username_algo(email):
 OIDC_OP_USER_ENDPOINT = config('OIDC_OP_USER_ENDPOINT')
 OIDC_RP_CLIENT_ID = config('OIDC_RP_CLIENT_ID')
 OIDC_RP_CLIENT_SECRET = config('OIDC_RP_CLIENT_SECRET')
+OIDC_RP_CLIENT_DOMAIN = config('AUTH0_DOMAIN')
```

Comment:
I guess this is ``OIDC_OP_DOMAIN``. Also please use ``config('OIDC_OP_DOMAIN`)`` for consistency. FYI this functionality does not yet exist in ``mozilla-django-oidc``. Let's merge it after that because we might have changes in the lib side.